services/src/main/scala/cromwell/pipeline/service/ProjectConfigurationService.scala

```diff
@@ -15,4 +15,7 @@ class ProjectConfigurationService(repository: DocumentRepository)(implicit ec: E
 
   def getById(projectId: ProjectId): Future[Option[ProjectConfiguration]] =
     repository.getByParam("projectId", projectId.value).map(_.headOption.map(fromDocument))
```

Comment:
We should not let the user get `ProjectConfiguration` if it was deleted, aren't we?

Reply:
Done